.github/CONTRIBUTING

```diff
@@ -0,0 +1,38 @@
+Contributing
```

Comment:
As long as we're writting some open source project, it's good to have a contributing guide :)